libraries/AP_AHRS/AP_AHRS_Backend.h

```diff
@@ -76,22 +75,10 @@ class AP_AHRS_Backend
     virtual int8_t get_primary_core_index() const { return -1; }
 
     // get the index of the current primary accelerometer sensor
-    virtual uint8_t get_primary_accel_index(void) const {
-#if AP_INERTIALSENSOR_ENABLED
-        return AP::ins().get_first_usable_accel();
-#else
-        return 0;
-#endif
-    }
+    virtual uint8_t get_primary_accel_index(void) const;
 
     // get the index of the current primary gyro sensor
-    virtual uint8_t get_primary_gyro_index(void) const {
```

Comment:
Simply un-inlining things requires a fair amount of testing, I suggest you don't do that in this PR